lib/api/policy.go

```diff
@@ -93,6 +93,10 @@ type PolicySpec struct {
 	// 	deployment != "dev"
 	// 	! has(label_name)
 	Selector string `json:"selector" validate:"selector"`
+
+	// Indicates, if True, that the rules in this policy should be applied before any data plane
+	// connection tracking, and that packets allowed by these rules should not be tracked.
```

Comment:
GoDoc should begin with the name of the field / method / etc that it is describing.

e.g "Untracked indicates whether or not the rules in this policy should go through connection tracking.  If True, ..."